pkg/collector/corechecks/snmp/config.go

```diff
@@ -105,8 +105,8 @@ func (c *snmpConfig) getStaticTags() []string {
 // toString used for logging snmpConfig without sensitive information
 func (c *snmpConfig) toString() string {
 	return fmt.Sprintf("snmpConfig: ipAddress=`%s`, port=`%d`, snmpVersion=`%s`, timeout=`%d`, retries=`%d`, "+
-		"user=`%s`, authProtocol=`%s`, privProtocol=`%s`, contextName=`%s`, oidConfig=`%#v`, metrics=`%#v`, "+
-		"metricTags=`%#v`, oidBatchSize=`%d`, profiles=`%#v`, profileTags=`%#v`, uptimeMetricAdded=`%t`",
+		"user=`%s`, authProtocol=`%s`, privProtocol=`%s`, contextName=`%s`, oidConfig=`%#v`, "+
```

Comment:
do `metrics` and `profiles` sensitive info? what about `profileTags` ?

Reply:
Neither of them are sensitive info, but removing them mainly for being too verbose and the info can be retrieved elsewhere (e.g. agent flare profiles yaml files).